torch/nn/quantized/modules/linear.py

```diff
@@ -61,3 +61,17 @@ def forward(self, x):
             self.out_scale,
             self.out_zero_point)
         return Y_q
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'weight'] = torch.ops.quantized.fbgemm_linear_unpack(destination[prefix + '_packed_weight'])
+        destination.pop(prefix + '_packed_weight')
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        self._packed_weight = torch.ops.quantized.fbgemm_linear_prepack(state_dict[prefix + 'weight'])
+        self.bias = state_dict[prefix + 'bias']
+        state_dict.pop(prefix + 'weight')
+        super()._load_from_state_dict(state_dict, prefix, local_metadata, False,
```

Comment:
given that the state_dict for linear is literally bias+weight, I don't think it makes sense to call parent method

Reply:
I can remove bias as well from state_dict. I saw there is a handling of hook in parent function, should that be called?

Reply:
there is 'output_scale' and 'output_zero_point' as well